optimade/validator/__init__.py

```diff
@@ -36,15 +36,16 @@ def validate():  # pragma: no cover
         default="http://localhost:5000/v0",
         help=(
             "The base URL of the OPTIMADE implementation to point at, "
-            "e.g. 'http://example.com/optimade/v1' or 'http://localhost:5000/v1"
+            "e.g. 'http://example.com/optimade/v1' or 'http://localhost:5000/v1'"
         ),
     )
     parser.add_argument(
         "-v",
         "--verbosity",
         action="count",
         default=0,
-        help="""Increase the verbosity of the output.""",
+        help="""Increase the verbosity of the output.
+        (-v: warning, -vv: info, -vvv: debug)""",
```

Comment:
This will add a lot of whitespace between the second and third line as well. Also, I'm not sure `argparse` actually renders the new-lines. In any case, if it does, you can instead do the following:
```suggestion
        help="Increase the verbosity of the output.\n(-v: warning, -vv: info, -vvv: debug)",
```

Reply:
With our current settings, argparse does not render the newline.  I could change the settings, but then we would have to manually insert new line characters in every line.

Reply:
Sounds like the best solution.

Reply:
FYI I think argparse has some vague idea of what terminal size it is outputting to when you run `--help`. I generated the docs by making my terminal quite thin before running `--help`